### ISSUE TITLE
Hijack failed on protocol switch

### ISSUE THREAD

**Issue by Dulanic** [sjtug/caddy2-filter#10]
Please see https://caddy.community/t/hijack-failed-on-protocol-switch/15635 and let me know if this is a problem with this plugin?

**Comment by skyzh** [comment]
Yes, I believe it is. This plugin doesn't support filtering contents in a web socket.

Considering switching websocket service to another domain / path so that it won't go through this plugin, or you may wait for us to have a fix on this 🥰
